### PR TITLE
Drop the dependency on python3-ntplib

### PR DESCRIPTION
**Pull request by poncovka**
* Use `chronyd -Q` to check the status of an NTP server.
* Don't synchronize the system time with a working NTP server.
* Let the chronyd service to synchronize the time.

### CODE REVIEW

File: pyanaconda/ntp.py [hunk 1]
```diff
@@ -107,25 +105,14 @@ def ntp_server_working(server_hostname, nts_enabled):
     :return: True if the given server is reachable and working, False otherwise
     :rtype: bool
     """
-    try:
-        # ntplib doesn't support NTS
-        if nts_enabled:
-            s = socket.create_connection((server_hostname, 4460), 2)
```
Comment (poncovka) [inline pyanaconda/ntp.py:113]: @mlichvar, can I replace your solution for nts with the one below? It seems to work fine.

Comment (mlichvar) [reply pyanaconda/ntp.py:113]: Yes, I think that's a good idea. Thanks.

Comment (mlichvar) [reply pyanaconda/ntp.py:113]: NETWORK_CONNECTION_TIMEOUT seems to be 46 seconds. Isn't that a bit too long for this check? I'd suggest something like 8 seconds, or so.

File: pyanaconda/ntp.py [hunk 2]
```diff
@@ -107,25 +105,14 @@ def ntp_server_working(server_hostname, nts_enabled):
...
-    except socket.error:
-        return False
+    timeout = str(NETWORK_CONNECTION_TIMEOUT)
+    directive = ["server", server_hostname, "iburst", "maxsamples", "1"]
+
+    if nts_enabled:
+        directive.append("nts")
 
-    return True
+    arguments = ["-Q", " ".join(directive), "-t", timeout]
```
Comment (jkonecny12) [inline pyanaconda/ntp.py:114]: I would remove the `timeout` variable instead just add the ``str(NETWORK_CONNECTION_TIMEOUT)`` directly.

Comment (poncovka) [reply pyanaconda/ntp.py]: Fixed.

File: pyanaconda/ntp.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ntp.py:113]: Yep, that is too long. The python3-ntplib library sets the timeout to 5 seconds, so I have changed it to the same number.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**